src/main/resources/static/mini-manage-api-stubs.yml

```diff
@@ -468,6 +468,35 @@ paths:
         500:
           $ref: '#/components/responses/500Response'
       x-codegen-request-body-name: body
+  /crn/search:
```

Comment:
I think I'd prefer `/person/search` here, as the `person` is the entity we're searching for